.github/workflows/publish.yml

```diff
@@ -42,16 +42,11 @@ jobs:
     runs-on: ubuntu-latest
     timeout-minutes: 15
     environment: production
+    permissions:
+      contents: write
```

Comment:
Does this only work for permissions for this repo? Like you'd still have to use github-app-token for making a release on another repo from this workflow?

Reply:
Yep, exactly -- it only affects the [permissions for the GITHUB_TOKEN](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token), which would only ever be for this repo